ldap/servers/slapd/pblock.c

```diff
@@ -492,6 +492,20 @@ slapi_pblock_get(Slapi_PBlock *pblock, int arg, void *value)
         (*(PRNetAddr **) value) = pblock->pb_conn->cin_addr_aclip;
         pthread_mutex_unlock(&(pblock->pb_conn->c_mutex));
         break;
+    case SLAPI_CONN_SERVERNETTYPE:
```

Comment:
"type" is a bit of a vague term. Perhaps `SLAPI_CONN_SERVERNETADDRESSFAMILY` would be better?